var/spack/repos/builtin/packages/drishti/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Drishti(PythonPackage):
+    """
+    Drishti is a command-line tool to guide end-users in optimizing I/O in their applications
+    by detecting typical I/O performance pitfalls and providing a set of recommendations.
+    """
+
+    homepage = "https://github.com/hpc-io/drishti"
+    git = "https://github.com/hpc-io/drishti"
+    pypi = "drishti-io/drishti-io-0.4.tar.gz"
+
+    maintainers = ["jeanbez", "sbyna"]
+
+    version("latest", branch="master")
+
+    version("0.4", sha256="bbbb272b4f6f44ae762f6cba28a2c589e15608691c559af0cc2f552590335d7b")
+
+    depends_on("darshan-util", type=("run"))
```

Comment:
Does this provide the same files as https://pypi.org/project/darshan/, or should we create a `py-darshan` package?

Reply:
No, actually, we that package as well. I created it and opened the PR #33430